File: gameboy-client/src/app/models/gameboy-metrics.ts
/**
 * Metrics sent from a GameBoy socket server.
 */
export class GameboyMetrics {

  /**
   * The average number of frames that were rendered on the server in the alst second.
   */
  framesPerSecond: number;

  /**
   * The total number of frames that had to be skipped due to the GPU falling behind the CPU.
   */
  skippedFrames: number;
}

File: Retro.Net.Api/RealTime/Models/GameBoyMetrics.cs
using MessagePack;

namespace Retro.Net.Api.RealTime.Models
{
    /// <summary>
    /// Metrics for a GameBoy socket client.
    /// </summary>
    [MessagePackObject]
    public class GameBoyMetrics
    {
        /// <summary>
        /// Gets or sets the frames per second.
        /// </summary>
        /// <value>
        /// The frames per second.
        /// </value>
        [Key("framesPerSecond")]
        public int FramesPerSecond { get; set; }

        /// <summary>
        /// Gets or sets the skipped frames.
        /// </summary>
        /// <value>
        /// The skipped frames.
        /// </value>
        [Key("skippedFrames")]
        public int SkippedFrames { get; set; }
    }
}
